sdk/storage/tests.yml

```diff
@@ -9,7 +9,7 @@ extends:
     Location: canadacentral
     CloudConfig:
       Preview:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+        SubscriptionConfiguration: $(sub-config-storage-test-resources-net)

```

Comment:
```suggestion
      Preview:
        SubscriptionConfigurations:
          - $(sub-config-azure-cloud-test-resources-preview)
          - $(sub-config-storage-test-resources-net)
```

Reply:
You still have to reference the base config unless you override ALL the config values (tenant id, service principal creds, etc.)